coremltools/test/neural_network/test_neural_networks.py

```diff
@@ -243,6 +243,36 @@ def test_classifier_with_string_label_list(self):
             predicted_feature_name='classLabel',
             class_labels=['1', '2', '3', '4', '5', '6', '7', '8', '9', '10'])
 
+    def test_classifier_without_class_labels(self):
+        coremltools.converters.tensorflow.convert(
+            self.frozen_graph_file,
+            mlmodel_path=self.converted_coreml_file,
+            input_name_shape_dict={'input': [1, 224, 224, 3]},
+            image_input_names=['input'],
+            output_feature_names=['Softmax'])
+
+    def test_classifier_nhwc(self):
+        # Test manually specifying the image format. The converter would have
+        # detected NHWC.
+        mlmodel = coremltools.converters.tensorflow.convert(
+            self.frozen_graph_file,
+            mlmodel_path=self.converted_coreml_file,
```

Comment:
this call pattern does not match with `coremltools.converters.tensorflow.convert`, it doesn't throw error because of `kwargs`. 
There seems to be a confusion: the arguments passed in here instead seem to be taken from [here](https://github.com/tf-coreml/tf-coreml/blob/1da1d0f45fca018c08119fd2e6e8e7583e0c7243/tfcoreml/_tf_coreml_converter.py#L529)